src/pages/Landing/SignupSteps/index.tsx

```diff
@@ -31,36 +28,127 @@ const StepsContainer = styled.div`
         </Heading>
+        <Subtitle>
+          <FormattedMessage
+            defaultMessage="Becoming a validator is a big responsibility with important
+          preparation steps. Only start the deposit process when you're ready."
+          />
+        </Subtitle>
       </ScrollAnimation>
       <StepsContainer>
         <ScrollAnimation animateIn="fadeInUp" animateOnce>
           <Step
-            imgUrl={ResponsibilityImgUrl}
-            title="1. Learn about your responsibilities"
-            content="The Eth2 upgrades will only be successful if validators understand the risks and responsibilities."
-          />
+            emoji="📚"
+            emojiAlt={intl.formatMessage({ defaultMessage: 'books' })}
+            title={intl.formatMessage({
+              defaultMessage: '1. Learn about your responsibilities',
+            })}
+            content={intl.formatMessage({
+              defaultMessage:
+                'The Eth2 upgrades will only be successful if validators understand the risks and responsibilities.',
+            })}
+          >
+            <Link to={routesEnum.FaqPage} primary withArrow>
+              <FormattedMessage defaultMessage="Validators FAQ" />
+            </Link>
+          </Step>
+        </ScrollAnimation>
+        <ScrollAnimation animateIn="fadeInUp" animateOnce delay={300}>
+          <Step
+            emoji="🔧"
+            emojiAlt={intl.formatMessage({ defaultMessage: 'wrench' })}
+            title={intl.formatMessage({ defaultMessage: '2. Prep nodes' })}
+            content={intl.formatMessage({
+              defaultMessage:
+                "You'll need to run an Eth1 and an Eth2 node to become a validator. Take a look at the checklist to prepare yourself and your equipment.",
+            })}
+          >
+            <Link to={routesEnum.checklistPage} primary withArrow>
+              <FormattedMessage defaultMessage="Hardware checklist" />
+            </Link>
+          </Step>
         </ScrollAnimation>
         <ScrollAnimation animateIn="fadeInUp" animateOnce delay={150}>
           <Step
-            imgUrl={GenerateKeyImgUrl}
-            title="2. Generate validator keys offline"
-            content="You need to generate new keys to identify and secure your validator."
-          />
+            emoji="🥋"
+            emojiAlt={intl.formatMessage({
+              defaultMessage: 'martial arts uniform',
+            })}
+            title={intl.formatMessage({
+              defaultMessage: '3. Practice on a testnet',
+            })}
+            content={intl.formatMessage({
+              defaultMessage:
+                'We recommend you go through the entire process on a testnet first to get comfortable.',
+            })}
+          >
+            <Link to={routesEnum.checklistPage}>
+              <FormattedMessage defaultMessage="Try the testnet [how can we dynamically link to the testnet?]" />
```

Comment:
@samajammin @wackerow ideally, I'd want to be able to dynamically link to !mainnet testnet with a variable here.

Example `<Link to={testnetNameVariable}.launchpad.ethereum.org>` – any ideas?